include/tvm/relay/type.h

```diff
@@ -214,55 +332,37 @@ class TypeRelation;
  */
 class TypeRelationNode : public TypeConstraintNode {
  public:
-  /*! \brief The name of the function */
```

Comment:
Could we have some kind of name field for debugging/printing purposes?

Reply:
The name is already in the EnvFunc